### PR TITLE
Update to JupyterLab 3.0

### LINKED ISSUE #100: Update for JupyterLab 3.0
Right now the documentation for JupyterLab points to a Binder link that installs 2.X, not 3.X!

### PR DESCRIPTION
**Pull request by jtpio**
Fixes #100 

For now commenting out some extensions that have not been updated yet.

Can be tested with this link: https://mybinder.org/v2/gh/jtpio/jupyterlab-demo/lab3?urlpath=lab

### CODE REVIEW

**Comment by jasongrout** [comment]
Would you mind pushing to a branch here on this repo, since this will be a semi-official somewhat long-lived branch that is updated as people update extensions? Then the binder link will read `jupyterlab/jupyterlab-demo` rather than `jtpio/jupyterlab-demo` and look more official.

**Comment by jtpio** [comment]
Sure. I'll need push access to the repo though.

**Comment by jtpio** [comment]
Looks like it's functional using the Binder link from above:

image

**Comment by jasongrout** [comment]
>  I'll need push access to the repo though.

Done

File: binder/environment.yml [hunk 1]
```diff
@@ -2,34 +2,34 @@ channels:
...
-- seaborn=0.9
+- seaborn=0.11
 - tensorflow=1.13
 - sympy=1.4
 - pyyaml
 - traittypes==0.2.1
 - invoke=1.2
 # C++ Kernel
-- xeus-cling=0.6
+# - xeus-cling=0.6
```
Comment (SylvainCorlay) [inline binder/environment.yml:28]: I think we can require `xeus-cling=0.10`, `xwidgets=0.24.2`, `xleaflet=0.13.1`  which should not cause conflicts.

**Comment by jtpio** [comment]
Closing in favor of #102 to use the upstream branch.

**Pull request closed without merging**